src/cljs/rems/dropdown.cljs

```diff
@@ -0,0 +1,52 @@
+(ns rems.dropdown
+  (:require [cljsjs.react-select]
+            [rems.guide-functions]
+            [rems.text :refer [text]])
+  (:require-macros [rems.guide-macros :refer [component-info example]]))
+
+;; TODO: add an ID to the input field and then link all labels to it
+(defn dropdown
+  "Single- or multi-choice, searchable dropdown menu."
+  [{:keys [items item->label item->selected? item->disabled? multi? on-change]
+    :or {item->selected? (constantly false)
```

Comment:
nit: wouldn't `item-selected?` and `item-disabled?` and `item-label` be as good?

Reply:
Done. Yeah, now that you mention it, I think they're better. The item->something naming came directly from the old component, and I sticked with it...